### PR TITLE
Update dind-builder to automatically deploy ECS service based on code produced by webapp

### PR DESCRIPTION
**Pull request by MichaelDeng03**
- Add functionality to deploy script so it: 
  - Creates a target group
  - Creates a listener rule for that target group
  - Starts a service hooked up with the existing balancer under truncated_arn.abop.ai

### CODE REVIEW

**Comment by kgang** [comment]
I reconciled the new file structure from main onto here (Just moved things under `demo_webapp/scripts/` and `docker/`. I also changed the ports from 5002 back to 5000, cuz I saw a reference to `EXPOSE 5000` in a dockerfile.

Please lmk if I broke something and I'll fix it

File: concrete/agents.py [hunk 1]
```diff
@@ -311,11 +316,36 @@ def deploy(self, backend_code: str, project_uuid: UUID):
         CMD ["flask", "run", "--host=0.0.0.0", "--port=80"]
         """
         )
-
+        start_script = dedent(
```
Comment (kgang) [inline concrete/agents.py:319]: Nice. Not sure how I feel about the inlined script, but the functionality is great

File: demo_webapp/scripts/deploy_to_aws.sh [hunk 2]
```diff
@@ -65,20 +92,102 @@ deploy_to_aws() {
     --query 'taskDefinition.taskDefinitionArn' \
     --output text)
 
-    echo "New task definition: $NEW_TASK_DEFINITION"
+    echo "Creating target group"
```
Comment (kgang) [inline demo_webapp/scripts/deploy_to_aws.sh:95]: Did you develop this script by using the inspect command through awscli?

**Review by kgang — APPROVED** [review]
LGTM

File: demo_webapp/scripts/deploy_to_aws.sh (hunk 2 above)
Comment (MichaelDeng03) [reply demo_webapp/scripts/deploy_to_aws.sh:95]: I manually launched a task via console, and then used describe-task-definition

File: concrete/agents.py (hunk 1 above)
Comment (MichaelDeng03) [reply concrete/agents.py:319]: Hmmm, I thought I moved that over to a script. I think it was overwritten in the main merge.

**Pull request merged**